src/OSPSuite.SimModelComp/src/SimModelComp_HelpFunctions.cpp

```diff
@@ -125,16 +125,16 @@ string Trim(const string & src)
 
 string StripFileName(const string & FilePath)
 {
-	int pos = FilePath.find_last_of(PATH_SLASH);
-	if(pos<0)
+	size_t pos = FilePath.find_last_of(PATH_SLASH);
+	if(pos == string::npos)
 		return FilePath;
```

Comment:
`string::npos` is usually defined as -1, but it's better to compare to `npos` instead of <0